bower.json

```diff
@@ -3,12 +3,13 @@
   "homepage": "https://github.com/garyb/purescript-strongcheck-laws",
   "description": "Law tests for core classes",
   "keywords": [
-    "purescript"
+    "purescript",
+    "pursuit-deprecated"
   ],
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/garyb/purescript-strongcheck-laws.git"
+    "url": "https://github.com/garyb/purescript-strongcheck-laws.git"
```

Comment:
The repository URL was updated to match the one in the [registry](https://github.com/purescript/registry/blob/master/bower-packages.json#L1246) so that a new version of the package can be published.